docs/_data/supporters.js

```diff
@@ -112,6 +113,16 @@ module.exports = async () => {
       {sponsors: [], backers: []}
     );
 
+  // Fetch images for sponsors and save their image dimensins
+  await Promise.all(
+    supporters.sponsors.map(async sponsor => {
+      for await (const chunk of needle.get(sponsor.avatar)) {
+        sponsor.dimensions = imageSize(chunk);
```

Comment:
nice.  yeah, they come down with a fixed height, but we don't know the width.

Reply:
Yup. HAving the width makes all the difference for being able to have the browser finalize the height of the section because it knows what space to reserve